.github/workflows/e2e.yml

```
@@ -0,0 +1,69 @@
+# Browser robot > opens browser > clicks around > checks if things work
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  e2e:
+    name: Playwright E2E
+    runs-on: ubuntu-latest
+    needs: []   # Run parallel with CI or use needs: [build-test]
```

actionlint (1.7.12)

[error] "needs" section should not be empty

(syntax-check)